src/assets/scss/_cards.scss

```diff
@@ -93,28 +98,112 @@
 
 }
 
-.card-tile {
+.module-tile {
```

Comment:
I'm not sure i get why we are removing the word card here when we use it everywhere else to describe the container

Reply:
"card" ended up being like... the whole wrapper shebang for each thing. Has the "Dig In" title, the body that holds the modules, and footer. I thought it was just a bit more clear, since there's also a .card-title, it made me a little more sane to not have both .card-tile and .card-title.

Reply:
I think today I'm gonna make a page in that coda we dont use making some definitions to go off so its easier for us to have these kinds of convos